htlcswitch/link.go

```diff
@@ -369,6 +376,10 @@ type channelLink struct {
 	// sub-systems with the latest set of active HTLC's on our channel.
 	htlcUpdates chan *contractcourt.ContractUpdate
 
+	// shutdownRequests is a channel that the channelLink will listen on to
+	// service shutdown requests from ShutdownIfChannelClean calls.
```

Comment:
nit: did you mean "serve"?

Reply:
I think they are synonyms in this case?

Reply:
do they? ok...thought we wanted a verb here😂